### DISCUSSION TITLE
Ability to charge players using deliverables.

### DISCUSSION THREAD

**Discussion by YTG2315** [gameserverapp/Platform#696] [Suggestions]
As per my last post. just separating the 2 :)

Also, currently as it stands, you can't charge someone using a deliverable, if that was possible. That would also be nice!

Additionally, it would have to charge the package cost amount. Im not to sure on if that would be possible since pricing may vary per pack.

**Comment by birdman3131** [comment]
Fairly sure you can deduct via sending negitive tokens.

Just not sure how to make sure the balance is greater than the deduct amount.

**Reply by YTG2315** [reply]
hey, i tried it, it does not let you do less than 1 as a deliverable specifically, i started making my shop /command based a while back however i realized and stopped, it only resurfaced today. Do let me know if you find a way though!

**Comment by mvaessen** [comment]
This should be available
